torch/csrc/autograd/function_hook.h

```diff
@@ -16,7 +16,7 @@ struct FunctionPreHook {
 
 struct FunctionPostHook {
   virtual ~FunctionPostHook() = default;
-  virtual variable_list operator()(const variable_list& grad_input, const variable_list& grad_output) = 0;
+  virtual variable_list operator()(const variable_list& outputs, const variable_list& inputs) = 0;
```

Comment:
I feel like it would still be good to retain the comment that outputs are usually referred to as grad_inputs, and inputs are grad_outputs.

Reply:
agreed, preserve that (like in the deleted comment below `/* grad_input */` and we're good to go.

Accepting conditional on adding the comments